assets/app/styles/_core.less

```diff
@@ -331,6 +331,59 @@ code {
   overflow-x: auto; // remove when we update to latest patternfly with twbs 3.3.4
 }
 
+.table-events {
+  background-color: #fff;
+  > tbody {
+    + tbody {
+      border-top-width: 1px;
+    }
+  }
+  tr {
+    @media (min-width: @screen-md-min) {
+      td[data-title="Name"] {
```

Comment:
Don't trigger off title text matches, mark the cell you want this behavior on. It's too easy to break this in the markup without realizing it by changing the title